index.html

```diff
@@ -1787,6 +1876,172 @@ <h2>
         </ol>
       </section>
     </section>
+    <section>
+      <h2>
+        File Handler Items
+      </h2>
+      <p>
+        Each <dfn data-local-lt="file handler's">file handler</dfn> represents
```

Comment:
Does ReSpec require the lt? Can you put the `'s` outside the link where used instead?

Reply:
I'm matching how it works for shortcut items in this spec here.